### PR TITLE
feat: Migrates to Gatsby V4

### PR DESCRIPTION
**Pull request by olivermrbl**
Upgrades the project to Gatsby V4.

Adds webpack config to include browserified versions of libraries `stream` and `crypto` that are used by `jsonwebtoken`. This is necessary because  webpack > 5 has removed polyfills that previously handled the dirty work for us.

Should be merged after #215

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,24 +43,20 @@
...
+    "stream-browserify": "^3.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.10.2",
     "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-backgrounds": "^5.3.19",
-    "@storybook/addon-links": "^5.3.19",
-    "@storybook/addons": "^5.3.19",
     "@storybook/react": "^5.3.19",
-    "autoprefixer": "^10.4.0",
```
Comment (kasperkristensen) [inline package.json:66]: PostCSS requires Autoprefixer [as seen here](https://tailwindcss.com/docs/guides/gatsby), so I don't think we should remove it 😃

Comment (olivermrbl) [reply package.json:66]: My bad. Added again :)

**Comment by pKorsholm** [comment]
lgtm

**Review by pKorsholm — COMMENTED** [review]
LGTM! 😄

**Pull request merged**